gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,7 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
+android.enableD8.desugaring=true
 distributionUrl=https\://services.gradle.org/distributions/gradle-4.7-all.zip
```

Comment:
Please make sure this also goes back to `bin`!

Reply:
Sorry, i didn't get what you said...

Reply:
This file must be reverted to look exactly like https://github.com/opendatakit/collect/blob/master/gradle/wrapper/gradle-wrapper.properties

In particular, notice that the one on `master` only loads binary files for gradle. The change made here set it to `all` which also includes documentation and other files.